src/sass/_fonts.scss

```diff
@@ -0,0 +1,4 @@
+@font-face {
+  font-family: 'SBL Hebrew';
+  src: url(../../assets/fonts/SBL_Hbrw.ttf);
```

Comment:
Maybe we can get or make a .woff for this?

Reply:
Done. update forthcoming.